src/styles/02-molecules/edition-card/_edition-card.scss

```diff
@@ -80,4 +80,8 @@
       margin-left: auto;
     }
   }
+
+  &--list {
+    margin: 30px;
```

Comment:
Instead of applying this on the list I imagine we'd want to build it into the card itself. We have helper functions for `margin-bottom` here (assuming that's the margin you want on this, which I think it is): https://github.com/NYPL/nypl-design-system/blob/development/src/styles/00-base/03-space/03-space-stack/_space-stack.scss#L21